docs/remote/troubleshooting.md

```diff
@@ -1068,6 +1072,26 @@ Extensions that access locally attached devices will be unable to connect to the
 
 ## Questions and feedback
 
+### Reporting quality issues
+
+If you run into an issue with one of the remote development extensions, it's important to collect the correct logs so that we'll be able to help [diagnose your issue](https://aka.ms/vscode-remote/issues/new).
+
+We recommend turning on trace level telemetry to get the most detailed logs: `F1` -> `Developer: Set Log Level...` -> `Trace`. From here, the type of logs depends on the context of your issue.
+
+You can get the Remote - SSH logs from the SSH Output channel: `F1` -> `Output: Focus on Output View`, and select "Remote - SSH" from the dropdown. Also please verify if you're able to SSH into your machine from an external terminal (not using Remote - SSH).
```

Comment:
There's one command `Remote-SSH: Show Log`

Reply:
Yes that should be included, good point. Updated to include `Remote-SSH: Show Log` so that it's consistent with the other extensions' show log commands. Still include the mention of the Remote - SSH output channel in case folks are uncertain if they're the same thing or prefer one method over the other.